github/info.json

```diff
@@ -7372,6 +7597,156 @@
         "server_url": ""
       },
       "parameters": []
+    },
+    {
+      "operation": "search_code",
+      "title": "Search Code",
+      "description": "Searches for specified query.",
+      "category": "investigation",
+      "annotation": "search_code",
+      "enabled": true,
+      "output_schema": {
+        "items": [
+          {
+            "sha": "",
+            "url": "",
+            "name": "",
+            "path": "",
+            "score": "",
+            "git_url": "",
+            "html_url": "",
+            "repository": {
+              "id": "",
+              "url": "",
+              "fork": "",
+              "name": "",
+              "owner": {
+                "id": "",
+                "url": "",
+                "type": "",
+                "login": "",
+                "node_id": "",
+                "html_url": "",
+                "gists_url": "",
+                "repos_url": "",
+                "avatar_url": "",
+                "events_url": "",
+                "site_admin": "",
+                "gravatar_id": "",
+                "starred_url": "",
+                "followers_url": "",
+                "following_url": "",
+                "organizations_url": "",
+                "subscriptions_url": "",
+                "received_events_url": ""
+              },
+              "node_id": "",
+              "private": "",
+              "html_url": "",
+              "keys_url": "",
+              "tags_url": "",
+              "blobs_url": "",
+              "forks_url": "",
+              "full_name": "",
+              "hooks_url": "",
+              "pulls_url": "",
+              "teams_url": "",
+              "trees_url": "",
+              "events_url": "",
+              "issues_url": "",
+              "labels_url": "",
+              "merges_url": "",
+              "archive_url": "",
+              "commits_url": "",
+              "compare_url": "",
+              "description": "",
+              "branches_url": "",
+              "comments_url": "",
+              "contents_url": "",
+              "git_refs_url": "",
+              "git_tags_url": "",
+              "releases_url": "",
+              "statuses_url": "",
+              "assignees_url": "",
+              "downloads_url": "",
+              "languages_url": "",
+              "milestones_url": "",
+              "stargazers_url": "",
+              "deployments_url": "",
+              "git_commits_url": "",
+              "subscribers_url": "",
+              "contributors_url": "",
+              "issue_events_url": "",
+              "subscription_url": "",
+              "collaborators_url": "",
+              "issue_comment_url": "",
+              "notifications_url": ""
+            }
+          }
+        ],
+        "total_count": "",
+        "incomplete_results": ""
+      },
+      "parameters": [
+        {
+          "title": "Search Query",
+          "name": "q",
```

Comment:
Please check if you can update this parameter name with some meaning full name for understanding purpose.

Reply:
Done